sympy/functions/elementary/complexes.py

```diff
@@ -465,9 +464,11 @@ def eval(cls, arg):
                     if base.func is cls and exponent is S.NegativeOne:
                         return arg
                     return Abs(base)**exponent
-                if base.is_positive == True:
-                    return base**re(exponent)
-                return (-base)**re(exponent)*exp(-S.Pi*im(exponent))
+                    if base.is_nonnegative:
```

Comment:
Done that, in reference to the original file.

````
if base.is_nonnegative:
         return base**re(exponent)
if base.is_negative:
         return (-base)**re(exponent)*exp(-S.Pi*im(exponent))
````

[This line] (https://github.com/sympy/sympy/blob/master/sympy/functions/elementary/complexes.py#L468l) is exactly below ``if exponent.is_integer:``